src/main/java/core/basesyntax/service/FileProcessor.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileProcessor {
```

Comment:
create FileReader interface

```suggestion
public class FileReaderImpl {
```